ynput/ayon-unreal
# Farm rendering changes

## Changelog Description
Changes to simplify logic for farm rendering.

## Additional review information

If you want to have reviews on farm, you need to add `unreal` with `.*` regex here: `ayon+settings://deadline/publish/ProcessSubmittedJobOnFarm/aov_filter` 

**Fixes:**
- refactored farm rendering flow
- review set on instance
- label in publisher showing folder path

## Testing notes:

> [!WARNING]
> ~~You need https://github.com/ynput/ayon-deadline/pull/49 and https://github.com/ynput/ayon-core/pull/958~~ been already merged

- create new rendering instance.
- set the render target for `Local Rendering`
- (render first the images)
- publish - it should publish as expected
- switch the rendering instance to `Farm Rendering`
- publish - you should have job on farm
